recipes/base_install.rb

```diff
@@ -236,7 +236,14 @@
 # Install FSx options
 include_recipe "aws-parallelcluster::_lustre_install"
 
-# Fetch EFA Driver
-unless node['cfncluster']['cfn_region'].start_with?("cn-")
-  include_recipe "aws-parallelcluster::_efa_fetch"
+# Install EFA
+if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) || node['platform'] == 'amazon' || (node['platform'] == 'ubuntu' && node['platform_version'] == "16.04")
+  unless node['cfncluster']['cfn_region'].start_with?("cn-")
+    include_recipe "aws-parallelcluster::_efa_install"
+  else
+    package 'openmpi-devel' do
```

Comment:
In Ubuntu there is no openmpi-devel, package name is different

Reply:
ubuntu will be a separate patch, I've removed it for now